src/field-element-creation-strategy.ts

```diff
@@ -53,7 +62,8 @@ export class FieldElementCreationStrategy implements FormElementCreationStrategy
     }
 
     private async fetchCountryPhoneCodes(): Promise<Map<string, string>> {
-        const apiUrl = `http://apilayer.net/api/countries?access_key=c90144e214d92738b0be5d6e0652e1e4&number=14158586273`;
+        const apiKey = process.env.API_KEY;
+        const apiUrl = `http://apilayer.net/api/countries?apiKey=${apiKey}`;
```

Comment:
to nalezy do pliku ENV wrzucic jakiegos tak jak chyba pisalem w ktoryms komentarzu